site/docs/tasks/configure-vault-plugin.md

```diff
@@ -0,0 +1,46 @@
+
+# Configure a new HashiCorp Vault Plugin
+
+The following steps show how to configure a new [Vault secret plugin](../concepts/vault_plugins.md) for Fybrik.
```

Comment:
maybe how to add a new Vault plugin? (here and in the task name)

Reply:
Fixed thanks